web/sites/default/config/core.entity_form_display.group.project.default.yml

```diff
@@ -533,6 +534,7 @@ content:
           options: ''
           popup: 0
         control: 0
+      path: '{"color":"#3388ff","opacity":"1.0","stroke":true,"weight":3,"fill":"depends","fillColor":"*","fillOpacity":"0.2","radius":"6"}'
```

Comment:
@radmosley , do you know where this change is coming from? It doesn't seem like a mapping integration should be happening in this change. 🤔